src/encoded/tests/test_file.py

```diff
@@ -135,12 +135,26 @@ def test_restricted_no_download(testapp, fastq_json):
     s3.put_object(Bucket='test-wfout-bucket', Key=resobj['upload_key'], Body=str.encode(''))
     download_link = resobj['href']
     testapp.get(download_link, status=307)
-    # fail download of restricted file (although with a 200 status?)
     testapp.patch_json(resobj['@id'], {'status': 'restricted'}, status=200)
+    # fail download of restricted file (although with a 200 status?)
```

Comment:
This comment I think refers to the 200 status above (result of patching to restricted?). It behaves correctly in that it now 403's the download I'd say, so parens can probably be removed.